src/services/AudiusBackend.js

```diff
@@ -2363,6 +2369,96 @@ class AudiusBackend {
       console.error(e)
     }
   }
+
+  /**
+   * Retrieves the claim distribution amount
+   * @returns {BN} amount The claim amount
+   */
+  static async getClaimDistributionAmount() {
+    await waitForLibsInit()
+    const wallet = audiusLibs.web3Manager.getWalletAddress()
+    if (!wallet) return
+
+    try {
+      const amount = await audiusLibs.Account.getClaimDistributionAmount()
+      return amount
+    } catch (e) {
+      console.error(e)
+      return null
+    }
+  }
+
+  /**
+   * Make the claim for the distribution
+   * NOTE: if the claim was already made, the response will 500 and error
+   * @returns {Promise<boolean>} didMakeClaim
+   */
+  static async makeDistributionClaim() {
+    await waitForLibsInit()
+    const wallet = audiusLibs.web3Manager.getWalletAddress()
+    if (!wallet) return
```

Comment:
Super minor but all these fns have an undefined return that makes them tricky to type properly, can we return either false or null in the no wallet case?